.dotnet/Azure.AI.OpenAI.Staging/Custom/AzureOpenAIClientOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenAI;
+
+namespace Azure.AI.OpenAI.Staging;
+
+/// <summary>
+/// Defines the scenario-independent, client-level options for the Azure-specific OpenAI client.
+/// </summary>
+public partial class AzureOpenAIClientOptions : OpenAIClientOptions
```

Comment:
Nothing to flag here, but as you evolve this forward, if you ever add mutable properties, be sure to use [AssertNotFrozen](https://learn.microsoft.com/en-us/dotnet/api/system.clientmodel.primitives.clientpipelineoptions.assertnotfrozen?view=azure-dotnet-preview#system-clientmodel-primitives-clientpipelineoptions-assertnotfrozen) for them.

Reply:
Much appreciated! Is this the right pattern to do it?
```csharp
    internal string Version
    {
        get
        {
            AssertNotFrozen();
            return _version;
        }
    }
    private readonly string _version;
```

Reply:
"Mutable" is definitely the key word and I discovered quickly why the assert on a `get` was a bad idea -- but I did leave it in the one place we currently still have a `set` and it seems to work as intended there.